chapters/hpmor-chapter-012.tex

```diff
@@ -200,8 +200,7 @@ \chapter{Impulse Control}
 
 But he \emph{could,} there was no rule saying he couldn’t, he \emph{could} just pretend he’d never heard that little whisper. Let the universe go on in exactly the same way it would have if that one critical moment had never occurred. Twenty years later, that was what he would desperately wish had happened twenty years ago, and twenty years before twenty years later happened to be right now. Altering the distant past was easy, you just had to think of it at the right time.
 
-Or…this was even \emph{more} counterintuitive…he could even
-inform, oh, say, \emph{Professor McGonagall,} instead of Draco \emph{or} Hermione. And she could get a few good people together and get that little extra spell taken off the Hat.
+Or…this was even \emph{more} counterintuitive…he could even inform, oh, say, \emph{Professor McGonagall,} instead of Draco \emph{or} Hermione. And she could get a few good people together and get that little extra spell taken off the Hat.
```

Comment:
Should the comma be outside the `\emph` in `\emph{Professor McGonagall,}`?

Reply:
done, but I assume there will be many more. I will try to include such a check in check-chapters.py / https://github.com/rrthomas/hpmor/pull/69